packages/workflow/src/TelemetryHelpers.ts

```diff
@@ -275,6 +276,11 @@ export function generateNodesGraph(
 					description: (node.parameters?.description as string) || '',
 				};
 			}
+
+			if (node.type === CHAIN_LLM_LANGCHAIN_NODE_TYPE) {
+				nodeItem.prompts =
+					(((node.parameters?.messages as IDataObject) || {}).messageValues as IDataObject[]) || [];
+			}
```

Comment:
Can we please find a way to avoid so many assertions? And use nullish coalescing instead of `||`? Also is there really a case where prompts will be an empty array?

Reply:
we had to use those assertions as parameter would not be included if value is default, yes by default in basic llm it is an empty array